terraform/projects/app-ecs-services/config/alerts/registers.yml

```diff
@@ -0,0 +1,9 @@
+groups:
+- name: Registers
+  rules:
+  - alert: RequestsExcess5xx
+    expr: sum(increase(requests{job="openregister-metric-exporter",status_range="5xx"}[5m])) by (app) >= 3
```

Comment:
Should we be using `rate` instead of `increase` here:
> increase should only be used with counters. It is syntactic sugar for rate(v) multiplied by the number of seconds under the specified time range window, and should be used primarily for human readability. Use rate in recording rules so that increases are tracked consistently on a per-second basis.

This is an open question, I am not sure I agree with the documentation. I like the fact that with this current check we can have a real number `3` which makes it a bit more readable (this is me assuming that it translates to we have had 3 5xx errors in the last 5 minutes).